launcher_scripts/conf/training/gpt3/175b.yaml

```diff
@@ -1,6 +1,6 @@
-hydra:
```

Comment:
These lines in the GPT config are actually quite relevant. They point to the config directory associated with nemo examples. There are some of the tp configs.
Leaving these lines below the defaults list will have following effect:
1. `tp_overlap@model.ub_tp_comm_overlap_cfg` will search for the tp config in the `conf/tp_overlap` directory
2. If it's not present then it will search in the `/opt/NeMo/examples/nlp/language_modeling/conf/tp_overlap`
3. If it's not found there it will default to null

It might happen in the future that you'd like to access the default set of nemo configs from within this repo and these lines might help you do that.

Reply:
Added those.